### ISSUE TITLE
-secret-env flag useless without -entrypoint

### ISSUE THREAD

**Issue by dplummer** [cruise-automation/daytona#14]
Is there any reason to have `-entrypoint` without `-secret-env`? Can `-secret-env` be removed and rolled into `-entrypoint` as the default behavior?

**Comment by broamski** [comment]
Some use cases involve running the binary in a stand alone fashion, not inside of a container, where the endpoint flag would not be used.

**Comment by dplummer** [comment]
So if you run `daytona -secret-env` without `-entrypoint` it modifies the existing environment with the secrets?

**Comment by broamski** [comment]
You're right! I think this might be a remnant from much older work that involved utilizing `eval` to capture outputs. Walking through this change, since it's just the removal of a boolean flag and since `-entrypoint` should always be also provided, this _shouldn't_ be a breaking change?
